internal/cmd/schema-registry/command_schema_describe.go

```diff
@@ -223,7 +224,18 @@ func printSchema(cmd *cobra.Command, schemaID int64, schema string, sType string
 	if sType != "" {
 		utils.Println(cmd, "Type: "+sType)
 	}
-	utils.Println(cmd, "Schema: "+schema)
+
+	if sType != "PROTOBUF" {
```

Comment:
protobuf is formatted already. The reason for `!=` is that avro schemas created using web ui and sometimes using cli, they have no schema type set which is `""`.